contribute/create-pull-request.md

```diff
@@ -43,6 +43,10 @@ Pull requests for Redux contributions must:
 - Not contain code that mutates state in reducers or thunks.
 - Not contain code that accesses the reducers state slice directly. Instead, the code should use state selectors to access state.
 
+Pull requests that add/modify unit test (Jest) must:
```

Comment:
Should it maybe be clarified that this applies to front end tests specifically?

Reply:
🤔 I think because is on the ### Frontend-specific guidelines section does not need to be specific

Reply:
Good point 👍